### ISSUE TITLE
Comments cannot be enabled from site editor on a page

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#54019]
### Description

When creating a page from the site editor, if the template used includes a comments block, then the comments block displays an enable comments button.

But you can't click that button because it's in the template. If you edit the template the comments block lists sample commments and the enable comments button - as it should - disappears.

Back in the page the button is back.

The document inspector does not have the discussion panel like the post editor does, so there is no way to turn comments on. 

### Step-by-step reproduction instructions

0. Use TT3 unmodified
1. Go to site editor
2. Click pages
3. Click the plus button in the top right of the sidebar
4. Give the page a name
5. Observe the enable comments button shown in the template

### Screenshots, screen recording, code snippet

https://github.com/WordPress/gutenberg/assets/107534/63020917-f75f-4073-a391-0350d6d5ddf4

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by SaxonF** [comment]
Thanks for highlighting! Definitely a gap we didn't account for. It looks like this issue is a duplicate of https://github.com/WordPress/gutenberg/issues/53188 which is part of https://github.com/orgs/WordPress/projects/127?pane=issue&itemId=37188610. 

The [suggestion](https://github.com/WordPress/gutenberg/issues/53188) is to just bring in the discussion panel which I agree would be a good first step. I imagine the "enable" button should just go away so when editing page you see the "comments are disabled" message and when editing template you see placeholder comments.

**Comment by draganescu** [comment]
I wouldn't close this one. https://github.com/WordPress/gutenberg/issues/53188 is about enabling comments from page editing in the site editor, this one is about the behavior of the comments block and of the template editing.

I think too that the enable comments button should go away - it's making the user do an action that - while it _seems_ contextual - is removed from the actual effect it causes.

**Comment by nudgeD** [comment]
I agree the “Enable comment” button doesn’t seem to serve the right purpose atm. I would suggest hiding it for now in the site editor.
